simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -56,8 +58,24 @@ public StandardExports() {
   public List<MetricFamilySamples> collect() {
     List<MetricFamilySamples> mfs = new ArrayList<MetricFamilySamples>();
 
-    mfs.add(new CounterMetricFamily("process_cpu_seconds_total", "Total user and system CPU time spent in seconds.",
-        osBean.getProcessCpuTime() / NANOSECONDS_PER_SECOND));
+    try {
+      /*
+      There are two interfaces com.ibm.lang.management.OperatingSystemMXBean and com.sun.management.OperatingSystemMXBean,
+      but no common interface which exposes getProcessCpuTime. Hence call on the implementing class, which is default access,
+      so use reflection hack to set it accessible.
+      */
+      Method method = osBean.getClass().getMethod("getProcessCpuTime", null);
+      if (!method.isAccessible()) {
+        method.setAccessible(true);
+      }
+      Long l = (Long) method.invoke(osBean);
+          mfs.add(new CounterMetricFamily("process_cpu_seconds_total","Total user and system CPU time spent in seconds.",
```

Comment:
Maybe github does not show whitespace updates correctly? Looks ok here: https://reviewable.io/reviews/prometheus/client_java/176 ?